optaplanner-docs/src/main/asciidoc/QuickStart/VaccinationScheduler/VaccinationSchedulerQuickStart-chapter.adoc

```diff
@@ -0,0 +1,275 @@
+
+.Priority rating table
+[cols="2,2,3", options="header"]
+|===
+|Age
+|Job
+|Priority rating
+
+|60
+|nurse
+|1060
+
+|33
+|nurse
+|1033
+
+|71
+|retired
+|71
+
+|52
+|office worker
+|52
+|===
+
+
+[id="optaplanner-solver-con_{context}"]
+== The OptaPlanner solver
+
+At the core of OptaPlanner is the solver, the engine that takes the problem data set and overlays the planning constraints and configurations. The problem data set includes all of the information about the people, the vaccines, and the vaccination centers. The solver works through the various combinations of data and eventually determines an optimized appointment schedule with people assigned to vaccination appointments at a specific center.  The following illustration shows a schedule that the solver created:
+
+ifdef::OPTAPLANNER-COMM[]
+image::QuickStart/VaccinationScheduler/vaccinationSchedulingValueProposal.png[]
+endif::OPTAPLANNER-COMM[]
+ifdef::OPTAPLANNER-ENT[]
+image::optimizer/vaccinationSchedulingValueProposal.png[]
+endif::OPTAPLANNER-ENT[]
+
+[id="continuous-planning-con_{context}"]
+== Continuous planning
+
+Continuous planning is the technique of managing one or more upcoming planning periods at the same time and repeating that process monthly, weekly, daily, hourly, or even more frequently. The planning window advances incrementally by a specified interval. The following illustration shows a two week planning window that is updated daily:
+
+ifdef::OPTAPLANNER-COMM[]
+image::QuickStart/VaccinationScheduler/vaccinationSchedulingContinuousPlanning.png[]
+endif::OPTAPLANNER-COMM[]
+ifdef::OPTAPLANNER-ENT[]
+image::optimizer/vaccinationSchedulingContinuousPlanning.png[]
+endif::OPTAPLANNER-ENT[]
+
+The two week planning window is divided in half. The first week is in the published state and the second week is in the draft state.  People are assigned to appointments in both the published and draft parts of the planning window. However, only people in the published part of the planning window are notified of their appointments. The other appointments can still change easily in the next run. Doing this gives OptaPlanner the flexibility to change the appointments in the draft part when you run the solver again, if necessary. For example, if a person who needs a second dose has a ready date of Monday and an ideal date of Wednesday, OptaPlanner does not have to give them an appointment for Monday if you can prove OptaPlanner can demonstrate that it can give them a draft appointment later in the week.
+
+You can determine the size of the planning window but just be aware of the size of the problem space. The problem space is all of the various elements that go into creating the schedule. The more days you plan ahead, the larger the problem space.
+
+
+[id="pinned-planning-entities-con_{context}"]
```

Comment:
the rest of the optaplanner docs defines ids differently.
If we merge this way, then all the other docs should do it this way too.